src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/MenuFlyoutTests/MenuFlyoutTests.cs

```diff
@@ -0,0 +1,146 @@
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.MenuFlyoutTests
+{
+	[TestFixture]
+	class MenuFlyoutTests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void MenuFlyoutItem_ClickTest()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuFlyoutItemTests.MenuFlyoutItem_Click");
+
+			_app.WaitForElement(_app.Marked("mfiButton"));
+
+			TakeScreenshot("Initial");
+
+			// step 1: press button to show menu
+			_app.FastTap(_app.Marked("mfiButton"));
+
+			TakeScreenshot("menuShown");
+
+			// step 2: click MenuFlyoutItem
+			_app.FastTap(_app.Marked("mfiItem"));
+
+			// step 3: check result
+			_app.WaitForText(_app.Marked("mfiResult"), "success");
+
+			TakeScreenshot("AfterSuccess");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Simple_MenuFlyout()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuBarTests.SimpleMenuBar");
+
+			_app.WaitForElement(_app.Marked("fileMenu"));
+			var result = _app.Marked("result");
+
+			TakeScreenshot("Initial");
+
+			void Validate(string topMenu, string item, string expectedResult)
+			{
+				_app.FastTap(_app.Marked(topMenu));
+
+				TakeScreenshot(item);
+
+				_app.FastTap(_app.Marked(item));
+
+				_app.WaitForText(result, expectedResult);
+			}
+
+			Validate("fileMenu", "exitMenu", "click text:Exit");
+			Validate("fileMenu", "openMenu", "click text:Open...");
+			Validate("fileMenu", "xamlUIMenu", "xamluicommand param:xamlUIMenu");
+
+			TakeScreenshot("AfterSuccess");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Simple_SubMenuFlyout()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuBarTests.SimpleMenuBar");
+
+			_app.WaitForElement(_app.Marked("fileMenu"));
+			var result = _app.Marked("result");
+
+			TakeScreenshot("Initial");
+
+			_app.FastTap(_app.Marked("fileMenu"));
+
+			TakeScreenshot("fileMenu");
+
+			_app.FastTap(_app.Marked("newMenu"));
+
+			TakeScreenshot("newMenu");
+
+			_app.FastTap(_app.Marked("plainTextMenu"));
+
+			_app.WaitForText(result, "command param:Plain Text");
+
+			TakeScreenshot("AfterSuccess");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Disabled_MenuFlyoutItem()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuBarTests.SimpleMenuBar");
+
+			_app.WaitForElement(_app.Marked("fileMenu"));
+			var result = _app.Marked("result");
+
+			TakeScreenshot("Initial");
+
+			_app.FastTap(_app.Marked("fileMenu"));
+
+			TakeScreenshot("fileMenu");
+
+			_app.FastTap(_app.Marked("disabledItem"));
+
+			TakeScreenshot("disabledItem");
+
+			Task.Delay(250);
+
+			_app.WaitForElement("disabledItem");
+
+			TakeScreenshot("AfterSuccess");
+		}
+
+		[Test]
+		[AutoRetry]
+		public void Dismiss_MenuFlyout()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.MenuBarTests.SimpleMenuBar");
+
+			_app.WaitForElement(_app.Marked("fileMenu"));
+			var result = _app.Marked("result");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'result'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)